src/uct/base/uct_iface.c

```diff
@@ -659,25 +653,23 @@ UCS_CLASS_DEFINE(uct_ep_t, void);
 
 static unsigned uct_iface_ep_error_handle_progress(void *arg)
 {
-    uct_base_ep_error_handle_info_t *err_info = arg;
+    uct_ep_h ep = arg;
     uct_base_iface_t *iface;
 
-    iface = ucs_derived_of(err_info->ep->iface, uct_base_iface_t);
-    iface->err_handler(iface->err_handler_arg, err_info->ep, err_info->status);
-    ucs_free(err_info);
+    iface = ucs_derived_of(ep->iface, uct_base_iface_t);
+    iface->err_handler(iface->err_handler_arg, ep, UCS_ERR_CONNECTION_RESET);
```

Comment:
rename the function to uct_iface_ep_conn_reset_handle_progress

Reply:
done